swo_aws_extension/flows/jobs/billing_journal/line_processors/support.py

```diff
@@ -0,0 +1,55 @@
+from typing import ClassVar
+
+from swo_aws_extension.constants import ItemSkusEnum, UsageMetricTypeEnum
+from swo_aws_extension.flows.jobs.billing_journal.line_processors.base_processor import (
+    GenerateItemJournalLines,
+)
+from swo_aws_extension.flows.jobs.billing_journal.line_validators.discount_validators import (
+    DefaultTrueDiscountValidator,
+    SupportDiscountValidator,
+)
+
+
+class GenerateSupportJournalLines(GenerateItemJournalLines):
+    """Generate journal lines for AWS support usage metrics."""
+
+    @property
+    def _item_sku(self):
+        return ItemSkusEnum.AWS_SUPPORT.value
+
+    @property
+    def _metric_id(self):
+        return UsageMetricTypeEnum.SUPPORT.value
+
+    _validator = SupportDiscountValidator
+    _exclude_services: ClassVar[list] = []
```

Comment:
Please always keep the same order: class variable first, then properties. Also, check the class above